htdocs/compta/facture/card.php

```diff
@@ -3071,7 +3071,7 @@
 
 	// Payment term
 	print '<tr><td class="nowrap fieldrequired">' . $langs->trans('PaymentConditionsShort') . '</td><td colspan="2">';
-	$form->select_conditions_paiements(isset($_POST['cond_reglement_id']) ? $_POST['cond_reglement_id'] : $cond_reglement_id, 'cond_reglement_id');
+	$form->select_conditions_paiements(isset($_POST['cond_reglement_id']) ? $_POST['cond_reglement_id'] : $cond_reglement_id, 'cond_reglement_id', -1, 1);
```

Comment:
It should be ok from proposal, but should not be for invoice because payments terms are a legal mandatory information for invoices in most countries in the world.

Reply:
ok; i've fixed it. @eldy